src/lib/QtWidgets/QAbstractScrollArea.ts

```diff
@@ -16,6 +14,8 @@ export type QAbstractScrollAreaSignals = QWidgetSignals;
 QAbstractScrollArea will list all methods and properties that are common to all scrollable widgets in the NodeGui world.
 
  */
+export type QAbstractScrollAreaSignals = QWidgetSignals;
```

Comment:
Wont moving this down associate the doc comments to this line instead of the class? @blncd2020